backend/src/main/java/com/example/cms/ticket/TicketController.java

```diff
@@ -67,4 +75,35 @@ public ResponseEntity<List<Response>> getTicketResponses(Pageable pageable, @Pat
                 httpHeaders,
                 HttpStatus.OK);
     }
+
+    @PutMapping("/{ticketId}")
+    public ResponseEntity updateTicketStatus(@PathVariable UUID ticketId ,@RequestBody TicketStatus ticketStatusToChangeTo) {
```

Comment:
Rozumiem, że to jest pod te przyciski typu "mark as resolved/irrelevant"?

Reply:
Zgadza się